include/albatross/src/indexing/group_by.hpp

```diff
@@ -111,6 +111,12 @@ template <typename KeyType, typename ValueType> class GroupedBase {
 
   ValueType first_value() const { return map_.begin()->second; }
```

Comment:
nit: maybe worth adding a comment here which says:
```
returns the value associated with the minimum key
```
and similar for `last_value`